core/common/src/main/java/alluxio/wire/MasterInfo.java

```diff
@@ -31,6 +32,8 @@
   private long mUpTimeMs;
   private String mVersion;
   private int mWebPort;
+  private String mZookeeperAddress;
+  private boolean mZookeeperEnabled;
```

Comment:
do we need to pass `mZookeeperEnabled`? Instead we could just check whether or not zookeeper address is set.

Reply:
That's one thing that I am curious about why do we need the property key :ZOOKEEPER_ENABLED?

Reply:
that's a good question, I don't see why we can't just check whether or not `alluxio.zookeeper.address` is empty. @gpang might have some insight